src/canopy-webpack-config.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-env node */
+const path = require('path');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const merge = require('webpack-merge')
+
+module.exports = function(name, overridesConfig) {
+  if (typeof name !== 'string') {
+    throw new Error('canopy-webpack-config expects a string name as the first argument')
+  }
+
+  if (typeof overridesConfig !== 'object') {
```

Comment:
I'm requiring an overridesConfig to encourage people to customize their webpack config to their needs